uint16 channel
uint16 pulse_width_in_microseconds
